src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -44,12 +65,7 @@ OpResult CondTraverseConsume(OpBase *opBase, QueryGraph* graph) {
 
```

Comment:
I think that `CondTraverse` ops are guaranteed to have only 1 child, but if that is not always true then we'll only properly traverse the first.

Reply:
guaranteed, the only operation which has more than one child is our Join operation.